centos7-desktop.json

```diff
@@ -8,7 +8,7 @@
   "iso_checksum": "6d44331cc4f6c506c7bbe9feb8468fad6c51a88ca1393ca6b8b486ea04bec3c1",
   "iso_checksum_type": "sha256",
   "iso_name": "CentOS-7-x86_64-DVD-1810.iso",
-  "iso_url": "https://mirrors.edge.kernel.org/centos/7.6.1810/isos/x86_64/CentOS-7-x86_64-DVD-1810.iso",
+  "iso_url": "https://mirrors.edge.kernel.org/centos/7.6.1810/isos/x86_64/CentOS-7-x86_64-DVD-1810.iso",",
```

Comment:
This part looks funky, probably should add the other lines back in?

Reply:
I guess it's because when I made the pull request it was in that state, now it's corrected, I'll revert the pull request and create a new one

Reply:
any advance or I need to revert the push request?